src/sonic-config-engine/portconfig.py

```diff
@@ -39,7 +39,8 @@
 INTF_KEY = "interfaces"
 OPTIONAL_HWSKU_ATTRIBUTES = ["fec", "autoneg"]
 
-BRKOUT_PATTERN = r'(\d{1,3})x(\d{1,3}G)(\[(\d{1,3}G,?)*\])?(\((\d{1,3})\))?'
+BRKOUT_PATTERN = r'(\d{1,6})x(\d{1,6}G?)(\[(\d{1,6}G?,?)*\])?(\((\d{1,6})\))?'
```

Comment:
>G? [](http://example.com/codeflow?start=37&length=2)

Why allow missing `G`? What is the default unit? Is there a document for the default unit?

Reply:
To allow to configure speed in megabytes:
```
                "1x100000[50G,40000,25000,10G]": ["Eth36"],
                "2x50G": ["Eth36/1", "Eth36/2"],
                "4x25G[10000]": ["Eth36/1", "Eth36/2", "Eth36/3", "Eth36/4"],
                "2x25000(2)+1x50G(2)": ["Eth36/1", "Eth36/2", "Eth36/3"],
                "1x50000(2)+2x25G(2)": ["Eth36/1", "Eth36/2", "Eth36/3"]
```